misc/areAnagrams.c

```diff
@@ -0,0 +1,53 @@
+#include <stdio.h>
+ 
+ //Function prototype
+int check_anagram(char [], char []);
+ 
+
+ //main
+int main()
+{
+  char a[100], b[100];
+ 
+  printf("Enter a string\n");
+  gets(a);
```

Comment:
[Why is the gets function so dangerous that it should not be used? - StackOveflow](https://stackoverflow.com/questions/1694036/why-is-the-gets-function-so-dangerous-that-it-should-not-be-used)